ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -222,10 +303,14 @@ def patch():
 
     wrap(RunnableSeq, "invoke", traced_runnable_seq_invoke(langgraph))
     wrap(RunnableSeq, "ainvoke", traced_runnable_seq_ainvoke(langgraph))
+    wrap(RunnableSeq, "astream", traced_runnable_seq_astream(langgraph))
```

Comment:
Do we need to do the same for `RunnableSeq.stream()`? Why only for async cases?

Reply:
Good question - while I was doing the fix for this and looking through their code, I didn't see a spot where `RunnableSeq.stream()` was used - `Pregel.stream()` calls `RunnableSeq.invoke()` under the hood. From what I've seen, `Pregel.astream()` has the only possibility of calling `RunnableSeq.astream()` under the hood if it was invoked from `astream_events`.

Happy to add the sync ones too but I'm not too sure how to test it in that case. Otherwise, I'm happy to add a comment explaining this!